src/Controllers/CheckoutController.php

```diff
@@ -106,34 +104,21 @@ public function __construct(
      * @throws \Twig_Error_Syntax
      */
     public function reinitPayment(
-        $orderId,
-        Twig $twig,
+        int $orderId,
```

Comment:
Are you sure that in no place NULL or something other than INT is given in here?

Reply:
Should be ok, order comes from 
 $eventDispatcher->listen('Ceres.LayoutContainer.MyAccount.OrderHistoryPaymentInformation', ReInitPaymentHook::class);
  $eventDispatcher->listen('Ceres.LayoutContainer.OrderConfirmation.AdditionalPaymentInformation', ReInitPaymentHook::class);